{"name": "test_001_ideal_use_case", "status": "broken", "statusDetails": {"message": "TypeError: comment() takes 1 positional argument but 2 were given", "trace": "@pytest.fixture\n    def pre_conditions():\n        \"\"\"Preconditions\"\"\"\n        Report.comment('***************Pre Conditions***************')\n        # - Change Timer 19 from 1 hour to 5 mins (300 secs)\n        # - :wycfg tmr[19] 300 0\n        # - :vycfg\n        Report.step('Timer 19 configured to 300 seconds.')\n        Report.comment('Command \":wycfg tmr[19] 300 0\" sent.')\n        Report.comment('Command \":vycfg\" sent.')\n        # - Turn on ignition, and wait for 30-60 seconds\n        Report.comment('Ignition turned On.')\n        Report.comment('Wait time of 50 seconds.')\n        sleep(0.5)\n        yield\n>       Report.comment('******************Tear Down*****************', True)\nE       TypeError: comment() takes 1 positional argument but 2 were given\n\nXirgo_KB1_Scripts\\Panic\\test_005_panic_cleared_by_timeout.py:23: TypeError"}, "attachments": [{"name": "stdout", "source": "e8b204c9-85a0-4465-8616-e3a567d70d4f-attachment.txt", "type": "text/plain"}], "start": 1637888382753, "stop": 1637888385770, "uuid": "9d9f0ccb-554c-4aca-8f44-37e8e96cda06", "historyId": "d0fb2f9bf5f72a295cc13c1598081a83", "testCaseId": "2fa0446fb174e0bba39f59166283f463", "fullName": "Xirgo_KB1_Scripts.Panic.test_005_panic_cleared_by_timeout#test_001_ideal_use_case", "labels": [{"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "Xirgo_KB1_Scripts.Panic"}, {"name": "suite", "value": "test_005_panic_cleared_by_timeout"}, {"name": "host", "value": "WX-AMADRID"}, {"name": "thread", "value": "19376-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Xirgo_KB1_Scripts.Panic.test_005_panic_cleared_by_timeout"}]}